rott/dukemusc.c

```diff
@@ -169,6 +172,20 @@ int MUSIC_Init(int SoundCard, int Address)
         return(MUSIC_Error);
     } // if
 
+    // add soundfonts
+    if (!MUSIC_SoundFonts_Set)
+    {
+        const char *soundfonts = Mix_GetSoundFonts();
```

Comment:
I'd prefer if we could call `Mix_GetSoundFonts()` to see if a soundfont path has been configured. If yes, use that. If not, provide a reasonable fallback like e.g. `/usr/share/soundfonts/default.sf2;/usr/share/sounds/sf2/default-GM.sf2`.

On Debian systems, for example, this will never be necessary, because it's guaranteed that a soundfont will always be available under the `/usr/share/sounds/sf2/default-GM.sf2` path and both SDL2_Mixer and Fluidsynth are configured to fall back to this path if no other soundfont has been explicitly selected.

Reply:
I wish the upstream `Mix_GetSoundFonts()` function was more thorough, it's supposed to "check common filesystem paths" but in practice in only checks [one path](https://github.com/libsdl-org/SDL_mixer/blob/SDL2/src/music.c#L1531-L1533) which I guess might be valid on Debian. On Arch the "preferred" soundfonts path that was told to me was `/usr/share/soundfonts/default.sf2`, which is not checked by SDL2. It also doesn't enumerate that directory, which I sorta assumed it would.